src/runtime_src/core/edge/user/shim.cpp

```diff
@@ -1523,6 +1523,8 @@ resetAIEArray(drm_zocl_aie_reset &reset)
 unsigned
 xclProbe()
 {
+  PROBE_CB;
```

Comment:
Please comment why PROBE_CB is not protected like all the other macro CBs?

Reply:
The whole "xclProbe" method is under #ifndef __HWEM__ macro. So, there is no need to add #ifndef __HWEM__ macro guard again for PROBE_CB.